From 951d8e14a4076e7eb999ec22347269bb53b2908c Mon Sep 17 00:00:00 2001
From: Don Zickus <dzickus@redhat.com>
Date: Tue, 16 Sep 2014 17:09:59 -0400
Subject: [usb] hub_handle_remote_wakeup() only exists for CONFIG_PM=y

Message-id: <1410887463-20674-528-git-send-email-dzickus@redhat.com>
Patchwork-id: 95391
O-Subject: [RHEL7 PATCH 527/591] usb: hub_handle_remote_wakeup() only exists for CONFIG_PM=y
Bugzilla: 821903
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>

https://bugzilla.redhat.com/show_bug.cgi?id=821903

commit 4a95b1fce97756d0333f8232eb7ed6974e93b054
Author: Stephen Rothwell <sfr@canb.auug.org.au>
Date:   Thu May 29 18:55:06 2014 +1000

    usb: hub_handle_remote_wakeup() only exists for CONFIG_PM=y

    Signed-off-by: Stephen Rothwell <sfr@canb.auug.org.au>
    Acked-by: Dan Williams <dan.j.williams@intel.com>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
index c7c1a4f..7fbdb17 100644
--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -3971,6 +3971,12 @@ EXPORT_SYMBOL_GPL(usb_disable_ltm);
 void usb_enable_ltm(struct usb_device *udev) { }
 EXPORT_SYMBOL_GPL(usb_enable_ltm);
 
+static int hub_handle_remote_wakeup(struct usb_hub *hub, unsigned int port,
+  u16 portstatus, u16 portchange)
+{
+ return 0;
+}
+
 #endif /* CONFIG_PM */
 
 
-- 
1.7.1